web-app/src/components/Commentary/data.js

```diff
@@ -49,19 +49,19 @@ function Data() {
   const handleChange = (panel) => (event, isExpanded) => {
```

Comment:
The component class can be named something more descriptive like Commentary.js
Also the data.js in the dictionary can be renamed to Dictionary.js